tools/docker/pandoc_makedocs.sh

```diff
@@ -56,7 +56,7 @@ echo "Create PDF"
     -o ${OUTPUT_BASE_NAME}-${LANGUAGE}.pdf $CHAPTERS \
     --verbose
 
-echo "Create epub"
+echo "Create mobi"
```

Comment:
is it ok mobi here and epub below in 62 and 66?

Reply:
Ah yea will d9